geowebcache/swiftblob/src/main/java/org/geowebcache/swift/SwiftBlobStore.java

```diff
@@ -183,14 +190,8 @@ public TileToKey(String coordsPrefix, MimeType mimeType) {
 
         @Override
         public String apply(long[] loc) {
-            long z = loc[2];
-            long x = loc[0];
-            long y = loc[1];
-            StringBuilder sb = new StringBuilder(coordsPrefix);
-
-            // builds the path to the tile
-            sb.append(z).append('/').append(x).append('/').append(y).append('.').append(extension);
-            return sb.toString();
+            // String in format "<prefix><z>/<x>/<y>.<ext>"
+            return String.format("%s%d/%d/%d.%s", coordsPrefix, loc[2], loc[0], loc[1], extension);
```

Comment:
This is [2 orders of magnitude slower than the previous code](https://stackoverflow.com/questions/44117788/performance-between-string-format-and-stringbuilder), but in the context of fetching a tile, the slowdown is most likely irrelevant.